src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.pushing(this);
+  }
+
+  static pushing(creature) {
+    this.alive.push(creature);
+  }
```

Comment:
```suggestion
    Animal.alive.push(this);
  }
```

You are using this operation only once so there's no need to create a method for it. You can remove your 'pushing' function